docs/iidm/model/shuntCompensator.md

```diff
@@ -19,18 +19,20 @@ The `com.powsybl.iidm.network.ShuntCompensator` interface is used to model a shu
 
 ## Section
 A section of a shunt compensator is an individual capacitor or reactor.
-A value of bPerSection positive means it is modeling a capacitor, an equipment that injects reactive
-power into the bus.
-A value of bPerSection negative means a reactor, an equipment that can absorb excess reactive power
-from the network.
+A specified susceptance and conductance are linked to each section of the shunt compensator.
+
+## Model
+There are two possible shunt compensator's models: linear and non linear. The attribute `model` describes the characteristics of the shunt compensator's distribution of susceptance
```

Comment:
Can you please add a generic sentence to say that a shunt compensator could be a capacitor or a reactor. For more details, see the model description. What do you think ?

Reply:
Done